pageserver/Cargo.toml

```diff
@@ -82,6 +82,7 @@ pem.workspace = true
 postgres_connection.workspace = true
 postgres_ffi.workspace = true
 pq_proto.workspace = true
+posthog_client_lite.workspace = true
```

Comment:
Let's sort the list here too (if you use vscode, you can select text (e.g. all lines starting with `p`) and then ctrl + shift + P and type `sort`)